src/components/atoms/InputRange/InputRange.tsx

```diff
@@ -0,0 +1,123 @@
+import React, { InputHTMLAttributes, useState, ChangeEvent, useRef, useEffect } from 'react';
+import styled, { css } from 'styled-components';
+import { colors } from '../../../constants/colors';
+import { calculateTrackPosition } from './helpers';
+
+const trackHeight = 8;
+const thumbDiameter = 40;
+
+const TrackStyles = css`
+  box-sizing: border-box;
+  border: none;
+  border-radius: ${trackHeight / 2}px;
+  height: ${trackHeight}px;
+  background: ${colors.neutral.light};
+`;
+
+const ThumbStyles = css`
+  box-sizing: border-box;
+  border: none;
+  border-radius: 50%;
+  width: ${thumbDiameter}px;
+  height: ${thumbDiameter}px;
+  background: #007468;
+`;
+
+const ThumbStylesFocus = css`
+  background: ${colors.base.secondary};
+`;
+
+interface IInput extends InputHTMLAttributes<HTMLInputElement> {
+  trackPosition: number;
```

Comment:
For a11y should we make `aria-label` required? 💅🏻

Reply:
I didn't want to put aria-label required because you have two options to solve this a11y issue:
- Add aria-label to `<InputRange />` => `<InputRange aria-label="slider" />`
- Add a proper label:
```
<div>
    <label htmlFor="slider">Slider</label>
    <InputRange id="slider" />
</div>
```